docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -65,6 +65,19 @@ Admins can also set the branch project to inherit values from a
 repo-level project settings configuration. This can be learned about at
 ['Using Repo Level Settings'](Repo-Level-Settings.md).
 
+### Project Flags
```

Comment:
![image](https://github.com/evergreen-ci/evergreen/assets/26798134/dde6f7c2-60bb-4c3f-b5d8-bbbb8aad14b8)

Reply:
I have a feeling that users would grep for things like "disable waterfall" or "disable cron builds". I think people might end up here because they don't realize that the repotracker settings is what they need. it might be a good idea to include those phrases somewhere so this bit comes up when that is searched.

Reply:
reworded a bit

Reply:
<img width="574" alt="image" src="https://github.com/evergreen-ci/evergreen/assets/26798134/2ae36ddd-82b6-415a-9b3a-00a73aa59871">

Reply:
thank you!